plugins/vpc-shared-eni/plugin/commands.go

```diff
@@ -169,5 +169,12 @@ func (plugin *Plugin) Del(args *cniSkel.CmdArgs) error {
 		log.Errorf("Failed to delete endpoint, ignoring: %v", err)
 	}
 
+	if netConfig.DeleteNetwork {
+		if err = nb.DeleteNetwork(&nw); err != nil {
```

Comment:
Please don't to function calls inside if statements. See DeleteEndpoint call above.